markets/perps-market/test/integration/Orders/OffchainAsyncOrder.referrer.test.ts

```diff
@@ -0,0 +1,226 @@
+import { ethers } from 'ethers';
```

Comment:
two tests:

- set fee collector to 0 and check nothing goes there
- set fee collector to more than 1 and check all fees sent there.
- set referral fee to more than 1, revert
- set referral fee to 0, revert